src/datta_lab_to_nwb/markowitz_gillis_nature_2023/markowitz_gillis_nature_2023behaviorinterface.py

```diff
@@ -1,22 +1,108 @@
 """Primary class for converting experiment-specific behavior."""
-from pynwb.file import NWBFile
-
+import numpy as np
+import pandas as pd
+from pynwb import NWBFile, TimeSeries
+from pynwb.behavior import (
+    BehavioralTimeSeries,
+    CompassDirection,
+    Position,
+    SpatialSeries,
+)
 from neuroconv.basedatainterface import BaseDataInterface
 
 class MarkowitzGillisNature2023BehaviorInterface(BaseDataInterface):
     """Behavior interface for markowitz_gillis_nature_2023 conversion"""
 
-    def __init__(self):
+    def __init__(self, filename: str):
```

Comment:
we are also getting in the habit of using Pydantic annotation typing for more specificity; `from pydantic import FilePath` instead of it just being a `str`